modules/ossm-federation-config-import.adoc

```diff
@@ -0,0 +1,134 @@
+
+The following example is for the `green-mesh` importing a service that was exported by `red-mesh`.
+
+.Example ImportServiceSet
+[source,yaml]
+----
+kind: ImportedServiceSet
+apiVersion: federation.maistra.io/v1
+metadata:
+  name: red-mesh #name of mesh that exported the service
+  namespace: green-mesh-system #mesh namespace that service is being imported into
+spec:
+  importRules: # first matching rule is used
+  # import ratings.bookinfo as ratings.bookinfo
+  - type: NameSelector
+    nameSelector:
+      importAsLocal: false
+      namespace: bookinfo
+      name: ratings
+      alias:
+        # services will be imported as <name>.mesh2-bookinfo.svc.mesh1-imports.local
+        namespace: bookinfo
+        name: ratings
+----
+
+.ImportServiceSet parameters
+[options="header"]
+[cols="l, a, a"]
+|===
+|Parameter |Description |Values
+|metadata:
+  name:
+|Name of the ServiceMeshPeer that exported the service to the federated mesh.
+|
+
+|metadata:
+  namespace:
+|Name of the ServiceMeshPeer namespace where the service is being imported.
+|
+
+|spec:
+  importRules:
+    -type:
+|Type of rule that will govern the import for the service. The first matching rule found for the service will be used for the import.
+|`NameSelector`, `LabelSelector`
+
+|spec:
+  importRules:
+    -type: NameSelector:
+      namespace:
+      name:
+|To create a `NameSelector` rule, specify the `namespace` of the service and the `name` of the service, as defined in the `Deployment` resource.
+|
+
+|spec:
+  importRules:
+    -type: NameSelector:
+      importAsLocal:
+|Set to `true` to aggregate remote endpoint with local services. When `true`, services will be imported as `<name>.<mesh-system>-<serviceName>.svc.cluster.local`
+|`true`/`false`
+
+|spec:
+  importRules:
+    -type: NameSelector:
+      importAsLocal:
+      namespace:
+      name:
+      alias:
+|To create a `NameSelector` rule that uses an alias for the service, after specifying the `namespace` and `name` for the service, then specify the alias for the `namespace` and the alias to be used for `name` of the service.
+|
+
+|spec:
+  importRules:
+    -type: LabelSelector
+|To create a `LabelSelector` rule, specify the `namespace` of the service and specify the `label` defined in the `Deployment` resource.
+|
+|===
+
+
+//PLEASE CHECK MY EXAMPLES
+
+.Import services with the name "ratings" from any mesh into blue-mesh
+[source,yaml]
+----
+kind: ImportedServiceSet
+apiVersion: federation.maistra.io/v1
+metadata:
+  name: *
+  namespace: blue-mesh-system
+spec:
+  importRules:
+  - type: NameSelector
+    nameSelector:
+      importAsLocal: false
+      namespace: bookinfo
+      name: ratings
+----
+
+.Import all services from the red-mesh's west-data-center namespace into the green-mesh and blue-mesh namespaces.
+[source,yaml]
+----
+kind: ImportedServiceSet
+apiVersion: federation.maistra.io/v1
+metadata:
+  name: red-mesh-system
+  namespace: green-mesh, blue-mesh
```

Comment:
Only one namespace may be specified and it must be the same namespace as the ServiceMeshPeer resource
```suggestion
  namespace: greeb-mesh-system
```